modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view_entries.jsp

```diff
@@ -137,8 +137,7 @@ DLViewEntriesDisplayContext dlViewEntriesDisplayContext = new DLViewEntriesDispl
 														<clay:link
 															cssClass="card-title text-truncate"
 															href="<%= dlViewEntriesDisplayContext.getViewFileEntryURL(fileEntry) %>"
-															label="<%= latestFileVersion.getTitle() %>"
-															title="<%= HtmlUtil.escapeAttribute(latestFileVersion.getTitle()) %>"
+															label="<%= HtmlUtil.unescape(latestFileVersion.getTitle()) %>"
```

Comment:
This looks suspicious. Are we double-escaping things somewhere?

Reply:
dlViewEntriesDisplayContext.getLatestFileVersion returns the latest escaped ( return fileVersion.toEscapedModel();) File version --> because of this return it's necessary to unescape title at jsp